### ISSUE TITLE
Can not open more templates at the same time

### ISSUE THREAD

**Issue by ollesmaps** [OpenOrienteering/mapper#963]
### Steps to reproduce
Can not open more templates at the same time.

### Actual behaviour
When selecting template, only one can be opened. You have to open one by one, separately.

### Expected behaviour
It would be good to open more templates at once. Sometimes there are many to be opened. It saves time.

### Configuration
OpenOrienteering Mapper 0.6.7

**Comment by dg0yt** [comment]
Couldn't this be quite confusing when Mapper needs to start asking questions about the templates?

**Comment by Zerbembasqwibo** [comment]
Good point.  OK it's not fun to open many, but you do it not very often. Let it be.
